src/ui/components/VPNCheckBoxRow.qml

```diff
@@ -19,7 +19,7 @@ RowLayout {
     property bool showDivider: true
     property var iconURL: ""
     property var leftMargin: 18
-    property var subLabelWrapMode: Text.WrapAnywhere
+    property var subLabelWrapMode: Text.WorldWrap
```

Comment:
Nice catch- thank you!! Just needs to be `Text.WordWrap` instead of `.WorldWrap`. It was added in #426 but should have been `Text.WordWrap`.